arm/Microsoft.RecoveryServices/vaults/backupPolicies/deploy.bicep

```diff
@@ -0,0 +1,31 @@
+@description('Required. Name of the Azure Recovery Service Vault')
+@minLength(1)
+param recoveryVaultName string
+
+@description('Required. Name of the Azure Recovery Service Vault Backup Policy')
+param policyName string 
+
+@description('Required. Configuration of the Azure Recovery Service Vault Backup Policy')
+param backupPolicyProperties object 
+
+@description('Optional. Customer Usage Attribution id (GUID). This GUID must be previously registered')
+param cuaId string = ''
+
+module pid_cuaId './.bicep/nested_cuaId.bicep' = if (!empty(cuaId)) {
+  name: 'pid-${cuaId}'
+  params: {}
+}
+
+resource vaultBackupPolicy 'Microsoft.RecoveryServices/vaults/backupPolicies@2021-08-01' = {
```

Comment:
Please rename main resource after its resource type. This is for this child and all other children and parent template.

e.g. in this case replace `vaultBackupPolicy` with `backupPolicy`

Reply:
Updated!